scyther_on_windows.md

```diff
@@ -0,0 +1,53 @@
+1. Install [MSYS2](https://www.msys2.org) and run it.
+2. Run the following command to install some tools for compilation:
+ “$ pacman -S mingw-w64-ucrt-x86_64-gcc”
+3. [from the msys2 terminal] Install the MingW cross compiler using the following command:
+ “pacman -S mingw-w64-cross-toolchain make cmake”
+ Note, when prompted to ‘Enter a selection(default = all): ’ just press enter to download everything.
+4. Install git using the command
+“pacman -S git”
+5. Clone the repository
+ “git clone https://github.com/cascremers/scyther.git”
+6. Download the dependencies
+ “pacman -S python3 flex bison”
+7. Run the build using the following command:
```

Comment:
Please check: running `make` from the main directory should just work here.

Reply:
In this case I prefer 'make' (this is also more future-proof in case we change the build process)

Reply:
In this case I would prefer `make`, because it is more future-proof.